frontend/dialob-composer-material/src/views/tree/NavigationTreeItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { ListItem, ListItemText, styled } from '@mui/material';
+import { TreeItem, ItemId } from '@atlaskit/tree';
+import { ArrowDropDown, ArrowRight } from '@mui/icons-material';
+import { DEFAULT_ICON_CONFIG } from '../../defaults/IconConfig';
+import { TreeDraggableProvided } from '@atlaskit/tree/dist/types/components/TreeItem/TreeItem-types';
+
+
+interface TreeItemProps {
+  item: TreeItem;
+  onExpand: (itemId: ItemId) => void;
+  onCollapse: (itemId: ItemId) => void;
+  provided: TreeDraggableProvided;
+}
+
+const PreTextIcon = styled('span')({
+  width: '24px',
```

Comment:
Check if this pixel value can cause problems when font size is changed by theme. Maybe it should be given with `em` unit which is dependent on the font size.